src/ai/backend/agent/config.py

```diff
@@ -32,6 +32,7 @@
         t.Key('event-loop', default='asyncio'): t.Enum('asyncio', 'uvloop'),
         t.Key('skip-manager-detection', default=False): t.ToBool,
         t.Key('aiomonitor-port', default=50102): t.Int[1:65535],
+        t.Key('allow-plugins', default=None): t.Null | tx.ToSet,
```

Comment:
How about renaming this to `allowed-compute-plugins` because this option only applies to `compute_plugin_ctx`?